administrator/language/de-DE/plg_schemaorg_article.ini

```
@@ -0,0 +1,29 @@
+; Joomla! German Translation
+; (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2024 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_ARTICLE="Schema.org - Artikel"
+PLG_SCHEMAORG_ARTICLE_DESCRIPTION_LABEL="Weitere Informationen über den Schema.org-Artikeltyp sind unter <a href='https://schema.org/Article' target='_blank' rel='noopener noreferrer'>https://schema.org/Article</a> zu finden."
```

```suggestion
PLG_SCHEMAORG_ARTICLE_DESCRIPTION_LABEL="Weitere Informationen über den Schema.org-Typ Artikel gibt es unter <a href='https://schema.org/Article' target='_blank' rel='noopener noreferrer'>https://schema.org/Article</a>"
```
vgl. andere Schema.org-Plugins